app/src/main/java/org/mozilla/rocket/extension/Extension.kt

```diff
@@ -0,0 +1,22 @@
+package org.mozilla.rocket.extension
```

Comment:
Would it be better to rename to something like ContextExtenstion or ContextExt, etc? Or this file is planned to contain extensions of all kinds of classes?

Reply:
updated to ContextExtension